pytest-profiling/pytest_profiling.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, svg):
         if os.path.isfile(self.gprof2dot):
             # Can't see gprof in the local bin dir, we'll just have to hope it's on the path somewhere
             self.gprof2dot = 'gprof2dot'
+        elif sys.version_info[0] == 2:
+            # We hope gprof2dot installed as python2 pip package
+            self.gprof2dot = 'python -m gprof2dot'
```

Comment:
Any reason why `sys.executable` is preferable? I prefer the readability of `sys.version_info.major` (as opposed to `sys.version_info[0]`

Reply:
Just for the record because I missed this earlier, the reason to avoid `'python'` is that the `python` executable could be a different one to the running version of python.  This is especially the case recently, when `python` is often an alias for Python 3.  The simple fix is
```diff
-self.gprof2dot = 'python -m gprof2dot'
+self.gprof2dot = sys.executable + ' -m gprof2dot'
```
But because `sys.executable` could contain spaces, and often does on Windows, it needs to be:
```diff
-self.gprof2dot = 'python -m gprof2dot'
+self.gprof2dot = '"{}" -m gprof2dot'.format(sys.executable)
```